.env-gh-codespaces-sample

```diff
@@ -0,0 +1,36 @@
+# This configuration adds the following overlays:
+#
+# docker-gh-codespaces: GitHub Codespaces settings (eg port 3000 exposed)
+#
+# docker-compose-publish-capybara-port.yml: expose the capybara port so a chrome driver
+#   running on the host machine can talk to the capaybara server and you can see the UI
+#   while the test are running.
+#
+# docker/dev/docker-compose-graphql.yml: start a graphql back-end server and
+#   a react-admin interface for managing administrative tasks
```

Comment:
We aren't really using the graphql stuff, and it takes up more memory. So I'd suggest keeping it disabled.